lexicon.txt

```diff
@@ -2451,6 +2456,9 @@ uartcommandconsole
 uartlite
 uarts
 ubasetype
+uc
+ucshadownamelength
```

Comment:
I think this would come after `ucsha` if you wanted to keep this list alphabetical? `sort -o lexicon.txt lexicon.txt`